### ISSUE TITLE
Make Mito work with Jupyter Lab Desktop

### ISSUE THREAD

**Issue by aarondr77** [mito-ds/monorepo#168]
**Describe the bug**
Mito doesn't work in the [JupyterLab App](https://github.com/jupyterlab/jupyterlab-desktop). 

Two users have reached out through Discord to tell us that this failed.

**Comment by naterush** [comment]
This might actually not be high effort...

**Comment by aarondr77** [comment]
Yeah, it just seems like one of those 'we don't know until we actually do it' and it could be weeks of work, things.

**Comment by aarondr77** [comment]
+1 another user asked for this
